packages/block-editor/src/components/font-appearance-control/index.js

```diff
@@ -98,12 +104,48 @@ export default function FontAppearanceControl( props ) {
 		style: { fontStyle: undefined, fontWeight: undefined },
 	};
 
+	// Build font style and weight options based on the font family faces.
+	let fontWeights = [];
+	let fontStyles = [];
+
+	fontFamilyFaces?.forEach( ( face ) => {
+		if ( face.fontWeight ) {
+			if (
+				fontWeights.findIndex(
+					( weight ) => weight.value === face.fontWeight
+				) === -1
```

Comment:
Could we do something a bit more idiomatic the conditions that are currently using `findIndex`? Such as:

```suggestion
				! fontWeight.some( ( weight ) => weight.value === face.fontWeight )
```

Reply:
Yes, nice idea! Done in https://github.com/WordPress/gutenberg/pull/61915/commits/a1bc596c1aad4622c0394cec9d75c763a582ace5.